### PR TITLE
feat: add a new more descriptive footer

### LINKED ISSUE #15: Improve Footer
### Current Situation
The current footer is underwhelming at not very informative.

The content of the footer can be extended and restyled it to be consistent with the continent and footer style found on layer5.io website.

### Desired Situation
Ideally, this doc site has the exact same for the content and functionality of the layer5.io website. Improve styling by take reference (and copying) from https://gitHub.com/layer5io/layer5.

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by 35C4n0r**
- The footer data and links are stored in hugo.toml
- styling has been added to _content_projects.scss.
- Breakpoints have been added.

**Notes for Reviewers**

This PR fixes #15 and #44 

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by 35C4n0r** [comment]
@leecalcote the PR is up for review.

**Comment by 35C4n0r** [comment]
@leecalcote ps: not a part of the issue #15, but should I add the code for the scrollbar too?
image

**Review by iArchitSharma — CHANGES_REQUESTED** [review]
@35C4n0r can you please change the logo svg to the one that is being used in header

**Comment by 35C4n0r** [comment]
> @35C4n0r can you please change the logo svg to the one that is being used in header

@iArchitSharma updated the logo

**Comment by leecalcote** [comment]
> > @35C4n0r can you please change the logo svg to the one that is being used in header
> 
> @iArchitSharma updated the logo

@ritiksaxena124 to confirm.

**Comment by leecalcote** [comment]
> @leecalcote ps: not a part of the issue #15, but should I add the code for the scrollbar too?
> image

Sounds great. Yes.

**Comment by 35C4n0r** [comment]
@leecalcote added the scrollbar :)

**Comment by ritiksaxena124** [comment]
> > @35C4n0r can you please change the logo svg to the one that is being used in header
> 
> @iArchitSharma updated the logo

@35C4n0r you are referencing the logo we have in the header, which is good. Could you please update the Twitter logo to "X"

**Comment by 35C4n0r** [comment]
Ok @ritiksaxena124 , keeping only the `https://layer5.io/subscribe`
But to include the X logo we would need to update to font awesome package to v6.4.2
Can you please guide me how to update this version.
I've tried running `go get github.com/FortAwesome/Font-Awesome@latest`, but i'm getting some error after that.

**Comment by ritiksaxena124** [comment]
> Ok @ritiksaxena124 , keeping only the `https://layer5.io/subscribe` But to include the X logo we would need to update to font awesome package to v6.4.2 Can you please guide me how to update this version. I've tried running `go get github.com/FortAwesome/Font-Awesome@latest`, but i'm getting some error after that.

Instead of using FontAwesome icons you can use the SVG

**Comment by leecalcote** [comment]
> > Ok @ritiksaxena124 , keeping only the `https://layer5.io/subscribe` But to include the X logo we would need to update to font awesome package to v6.4.2 Can you please guide me how to update this version. I've tried running `go get github.com/FortAwesome/Font-Awesome@latest`, but i'm getting some error after that.
> 
> Instead of using FontAwesome icons you can use the SVG

Yes, please.

**Comment by 35C4n0r** [comment]
@ritiksaxena124 replaced fa with the SVGs

**Review by iArchitSharma — CHANGES_REQUESTED** [review]
@35C4n0r well done with the logo
also add space b/w `2023 Copyright` and shift **Privacy** and **Terms** in right side just as it is in layer5.io
image

**Comment by saurabh100ni** [comment]
Please add this as your agenda in [Doc](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.69qskjv558bk) to present it in the Website Meet today at 6:30 pm IST/7:00 am CT

**Comment by 35C4n0r** [comment]
> layer5.io



> @35C4n0r well done with the logo also add space b/w `2023 Copyright` and shift **Privacy** and **Terms** in right side just as it is in layer5.io image

@iArchitSharma fixed this.

**Comment by iArchitSharma** [comment]
> add space b/w `2023 Copyright`

nice but you didn't added space b/w **2023** and **Copyright**

**Comment by 35C4n0r** [comment]
@iArchitSharma wierd bug, but it looks fine on my local
image

I'll see how i can fix this

**Comment by 35C4n0r** [comment]
@iArchitSharma there we go, string concatenation works.

**Review by iArchitSharma — APPROVED** [review]
@35C4n0r Good job, LGTM //
@ritiksaxena124

**Review by leecalcote — COMMENTED** [review]
Thank you, @35C4n0r, @iArchitSharma.

Will you move the footer css into a separate .scss file, and import into _content_project.scss?

**Comment by 35C4n0r** [comment]
@leecalcote done :)

File: layouts/partials/footer.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+                <a href="{{ .url  }}" class="{{ .class }}" target="_blank" rel="noreferrer">
+                    <img height="30px"  src="{{ .icon | safeURL }}">
+                </a>
+                {{ end }}
+            </div>
+        </div>
+        <div class="row footer-info">
+            <div class="col-lg-3 footer-desc">
+                <p>An empowerer of engineers, Layer5 helps you extract more value from your infrastructure. Creator and
+                    maintainer of cloud naative standards. Maker of Meshery, the cloud native manager.</p>
```
Comment (ritiksaxena124) [inline layouts/partials/footer.html:21]:
```suggestion
                    maintainer of cloud native standards. Maker of Meshery, the cloud native manager.</p>
```

File: layouts/partials/footer.html [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+                </div>
+                <div class="row footer-info2 subscribe">
+                    <div class="footer-end">
+                        <form name="contactform">
+                            <div>
+                                <p>
+                                    Subscribe to our Newsletter
+                                </p>
+                                <input class="footer-input" type="email" placeholder="Email Address"/>
+                                <button class="footer-button" title="Subscribe" type="submit">Subscribe</button>
```
Comment (ritiksaxena124) [inline layouts/partials/footer.html:66]: Change the text color to white 
Image from layer5.io
image

Image from deploy preview of your PR
image

File: assets/scss/footer.scss [hunk 3]
```diff
@@ -0,0 +1,502 @@
...
+      display: none;
+    }
+
+    tr {
+      display: flex;
+      box-sizing: border-box;
+      padding: 1em;
+      margin-bottom: 2.5em;
+      border: none;
+      background: #fbfbfd;
```
Comment (ritiksaxena124) [inline assets/scss/footer.scss:103]: Use the variables for colors, if there is no entry will you make a new one

**Review by ritiksaxena124 — CHANGES_REQUESTED** [review]

File: layouts/partials/footer.html (hunk 1 above)
Comment (35C4n0r) [reply layouts/partials/footer.html:21]: Probably need changes too.
https://github.com/layer5io/layer5/blob/61a55eea2cfd0a56eae65c2e0c57efb292881418/src/assets/discuss/html/footer.html#L38

**Review by leecalcote — APPROVED** [review]
Excellent, @35C4n0r 🥇

**Comment by leecalcote** [comment]
Ideally, in all of our site's footers, we'll using SVGs with vectors inside (not SVGs with rasterized PNGs inside).

File: assets/scss/footer.scss [hunk 4]
```diff
@@ -0,0 +1,503 @@
...
+  .search-widget {
+    display: none;
+  }
+
+  .custom-search-banner {
+    margin: -1em 0 2em 0;
+    background-color: $primary;
+
+    .custom-search-banner-wrap {
+      max-height: 250px;
```
Comment (coder12git) [inline assets/scss/footer.scss:46]: Please use `rem` instead of `px`

File: hugo.toml [hunk 5]
```diff
@@ -177,38 +177,126 @@ no = 'Sorry to hear that. Please <a href="https://github.com/layer5io/docs/issue
...
...D0iMjA0IiB5PSI1Mi4zIiBjbGFzcz0ic3QwIi8+PHJlY3Qgd2lkdGg9IjQ3LjUiIGhlaWdodD0iNDIuNyIgeD0iMjA0IiB5PSIxMDQuMSIgY2xhc3M9InN0MCIvPjxyZWN0IHdpZHRoPSI0Ny41IiBoZWlnaHQ9IjQzLjIiIHg9IjE0Ny45IiB5PSI1Mi4zIiBjbGFzcz0ic3QwIi8+PHJlY3Qgd2lkdGg9IjQ3LjUiIGhlaWdodD0iNDIuNyIgeD0iMjYwLjIiIHk9IjEwNC4xIiBjbGFzcz0ic3QwIi8+PHJlY3Qgd2lkdGg9IjQ3LjUiIGhlaWdodD0iNDMuNiIgeD0iMjA0IiBjbGFzcz0ic3QwIi8+PHJlY3Qgd2lkdGg9IjQ3LjUiIGhlaWdodD0iNDMuMiIgeD0iOTEuNyIgeT0iNTIuMyIgY2xhc3M9InN0MCIvPjxyZWN0IHdpZHRoPSI0Ny41IiBoZWlnaHQ9IjQyLjciIHg9IjE0Ny45IiB5PSIxMDQuMSIgY2xhc3M9InN0MCIvPjxwYXRoIGQ9Ik00NTAuNCwxMjQuMWMtMTAuNC03LTM0LjItOS41LTUyLjUtNmMtMi40LTE3LjItMTItMzIuMy0yOS41LTQ1LjhsLTEwLjEtNi43bC02LjcsMTAuMSBjLTEzLjIsMjAtMTYuOCw1Mi45LTIuNyw3NC42Yy02LjMsMy40LTE4LjUsOC0zNC44LDcuN0gxLjhjLTYuMywzNi41LDQuMiw4My45LDMxLjYsMTE2LjVDNjAsMzA2LDEwMCwzMjIuMSwxNTIuMywzMjIuMSBjMTEzLjEsMCwxOTYuOS01Mi4xLDIzNi0xNDYuOGMxNS40LDAuMyw0OC42LDAuMSw2NS42LTMyLjVjMS4xLTEuOCw0LjctOS41LDYuMS0xMi4zTDQ1MC40LDEyNC4xeiIgY2xhc3M9InN0MCIvPjxwb2x5Z29uIHBvaW50cz0iODMuMiAxMDQuMSA4My4xIDEwNC4xIDM1LjcgMTA0LjEgMzUuNyAxNDYuOCA4My4yIDE0Ni44IiBjbGFzcz0ic3QwIi8+PHJlY3Qgd2lkdGg9IjQ3LjUiIGhlaWdodD0iNDIuNyIgeD0iOTEuNyIgeT0iMTA0LjEiIGNsYXNzPSJzdDAiLz48L2c+PC9nPjwvZz48L3N2Zz4="
```
Comment (coder12git) [inline hugo.toml:220]: Use SVGs with vectors, you can directly  import those icons SVGs in `assets/images` directory and can add a link here.

**Review by coder12git — CHANGES_REQUESTED** [review]

**Comment by coder12git** [comment]
@35C4n0r footer is not completely responsive.

Screenshot 2023-11-09 082527

Ideally it should look same, in mobile view as we have for `layer5.io` sites, check below screenshots:

Screenshot 2023-11-09 082628
Screenshot 2023-11-09 082609

**Comment by leecalcote** [comment]
Hang in there, @35C4n0r. You're doing some good work.

File: assets/scss/footer.scss (hunk 4 above)
Comment (35C4n0r) [reply assets/scss/footer.scss:46]: Do i need to replace all the px with rems??

**Comment by 35C4n0r** [comment]
> @35C4n0r footer is not completely responsive. Also the description text and some other texts used here is 'open-sans' instead of 'Qanleas Soft`.
> 
> Screenshot 2023-11-09 082527
> 
> Ideally it should look same, in mobile view as we have for `layer5.io` sites, check below screenshots:
> 
> Screenshot 2023-11-09 082628 Screenshot 2023-11-09 082609

@coder12git made the css more, responsive will be pushing the code soon.

File: hugo.toml (hunk 5 above)
Comment (35C4n0r) [reply hugo.toml:220]: Sure.

**Comment by 35C4n0r** [comment]
Pushed the code, @coder12git kidly give a re-review

File: assets/scss/_variables_project.scss [hunk 6]
```diff
@@ -67,6 +67,7 @@ $web-font-path: "https://fonts.googleapis.com/css?family=#{$google_font_family}&
 $font-awesome-font-name: "Font Awesome 6 Free" !default;
 
 $td-fonts-serif: "Qanelas Soft", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+$font-qanelas: "Qanelas Soft", sans-serif, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial,  "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
```
Comment (coder12git) [inline assets/scss/_variables_project.scss:70]:
```suggestion
$font-qanelas: "Qanelas Soft", "Open Sans", sans-serif, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial,  "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
```

**Comment by coder12git** [comment]
@35C4n0r great work you did. Everything looks good to me, just remove that extra spacing from bottom (in mobile view).
Screenshot 2023-11-11 195925

**Comment by 35C4n0r** [comment]
@coder12git @coder12git apparently it wasn't a bug related to footer, maybe background overflow https://github.com/layer5io/docs/pull/33 ?? I've fixed it now.

**Comment by iArchitSharma** [comment]
@35C4n0r good work there, was the scrollbar supposed to be that way?

**Comment by abhijeetgauravm** [comment]
Hey @35C4n0r  Let's discuss this on website call tomorrow at 6:30 PM IST / 7 AM Central time. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima).

**Review by coder12git — APPROVED** [review]

**Comment by coder12git** [comment]
@35C4n0r thanks, @leecalcote this PR is good to merge now 🚀

**Comment by 35C4n0r** [comment]
Nope, that is because of this https://github.com/layer5io/docs/pull/46#issuecomment-1806875029
will need to discuss a better way to resolve this, that overflow in not a part of HTML??? Most probably because of background image, not sure.
image

**Comment by leecalcote** [comment]
Thanks for sticking with this, @35C4n0r

**Pull request merged**